src/utils/jsonlogic.ts

```diff
@@ -0,0 +1,169 @@
+/**
+ * JsonLogic type checking utility functions
+ */
```

Comment:
maybe a `@todo` is in order to figure out a better place for these type definitions? I regrettably must admit that using a monorepo for all our TS would probably make sense to put code in logical places while still being able to import/package it to well-scoped packages. Having this OF backend specific stuff in something that pretends to talk "only" formio concepts is blurring the lines.

Reply:
What other components would make use of this util module?

Reply:
we want to use TS for our admin-builder code too, which is currently webpack/JS. So these type definitions will become relevant for the backend project. The whole service fetch configuration modal/form components would benefit from these types.

Reply:
Ok I see, admin-builder == the current js in the django backend I suppose